src/import.ml

```diff
@@ -253,3 +253,31 @@ module Ocaml_version = struct
 
   include Ocaml_version
 end
+
+exception User_error of string
+
+module Handlers = struct
+  let unpwrap = function
+    | `Ok () -> ()
+    | `Error err_msg -> show_message `Error "%s" err_msg
+  ;;
+
+  let w1 f x =
+    try `Ok (f x) with
+    | User_error e -> `Error e
+  ;;
+
+  let ws f x y =
+    match f x with
+    | `Ok f' ->
+      (try `Ok (f' y) with
+       | User_error e -> `Error e)
+    | `Error e -> `Error e
+  ;;
+
+  let w2 f = ws (w1 f)
+  let w3 f x = ws (w2 f x)
+  let w4 f x y = ws (w3 f x y)
+  let w5 f x y z = ws (w4 f x y z)
+  let _w6 f x y z w = ws (w5 f x y z w)
+end
```

Comment:
Are these standard names? I'm having trouble understanding what `ws` and `wX` mean?

Reply:
I just move it from elsewhere, and I agree it's a bit puzzling. We didn't use it in the other commands anyway, I will probably move to do the same.

Reply:
`wrap` I guess